package.json

```diff
@@ -34,10 +39,11 @@
       "last 1 chrome version",
       "last 1 firefox version",
       "last 1 safari version"
-	]
+    ]
   },
   "devDependencies": {
+    "eslint": "^8.7.0",
```

Comment:
Rreate-react-app comes with eslint config. Maybe there is no need to include eslint in our dev dependencies too?

Reply:
In our previous conversation you said that maybe I haven't installed eslint prettier. I double-checked and saw in Prettier ESlint prerequisites that I need to install prettier and eslint packages. [ESlint Prettier ReadMe](https://github.com/idahogurl/vs-code-prettier-eslint/blob/master/README.md). Do you think I should remove it?

Reply:
Yes. These packages (eslint in this case) must be installed either locally or globally. The reate-react-app comes with them as a dep. So no need to manually install them or add them in here. So please remove eslint (`npm uninstall eslint --save-dev`).